source/extensions/health_checkers/thrift/utility.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "envoy/config/core/v3/health_check.pb.h"
+#include "envoy/extensions/health_checkers/thrift/v3/thrift.pb.h"
+#include "envoy/extensions/health_checkers/thrift/v3/thrift.pb.validate.h"
+
+#include "source/common/config/utility.h"
+#include "source/common/protobuf/protobuf.h"
+#include "source/common/protobuf/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HealthCheckers {
+namespace ThriftHealthChecker {
+
+namespace {
+
+static const envoy::extensions::health_checkers::thrift::v3::Thrift
+getThriftHealthCheckConfig(const envoy::config::core::v3::HealthCheck& health_check_config,
+                           ProtobufMessage::ValidationVisitor& validation_visitor) {
```

Comment:
nit: feels like this just belongs in config.cc?

Reply:
This is also used in thrift_test.cc. Extracting it to a separated file can minimize the link overhead and dependency.